Makefile

```diff
@@ -116,7 +116,8 @@ config: certs ## Set up default config
 
 .PHONY: musl
 musl: ## Add target for musl
-	rustup target add $(uname_m)-unknown-linux-musl
+	rustup target list | grep -qc '$(uname_m)-unknown-linux-musl' || \
```

Comment:
The fix to making it a string was correct, but the string itself is a little off. Installed targets (what we want grep to look for) end in " (installed)"